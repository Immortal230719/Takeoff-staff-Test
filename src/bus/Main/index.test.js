import React from 'react';
import { shallow } from 'enzyme';
import { Main } from './index';

describe('Main', () => {
  const component = shallow(<Main disabled className="center">Layout</Main>);

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
