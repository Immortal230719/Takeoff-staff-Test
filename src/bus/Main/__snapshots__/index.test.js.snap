// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render correctly 1`] = `
<Styled(MuiBox)
  className="makeStyles-root-1"
>
  <Link
    className="makeStyles-links-2"
    to="/login"
  >
    <LoginBtn
      onClick={[Function]}
    >
      Login
    </LoginBtn>
  </Link>
  <Link
    className="makeStyles-links-2"
    to="/registration"
  >
    <SignUpBtn
      onClick={[Function]}
    >
      Sign Up
    </SignUpBtn>
  </Link>
</Styled(MuiBox)>
`;
